### ISSUE TITLE
should @trust be moved?

### ISSUE THREAD

**Issue by melvincarvalho** [solid/solid-cli#2]
dont mean to be a pain, but I was trying to do some debugging in this code, and want to make sure I'm looking at the most recent code base (as a learning experience)

https://github.com/solid/solid-cli/blob/master/src/SolidClient.js#L4

has @trust moved to @solid ?

**Comment by kjetilk** [comment]
I think so, but @justinwb would be the authority on this, I think.

**Comment by RubenVerborgh** [comment]
Yes, repo predates that move. Should upgrade.

**Comment by justinwb** [comment]
Yeah so we moved most (not all) of the @trust dependencies over to the @solid namespace - but this one was included. I believe this is the full list:

jose
keychain
oidc-op
oidc-rp
oidc-rs

**Comment by kjetilk** [comment]
I'm getting bitten by this because there is a bunch of security warnings from `base64url`, and this appears to be the one that is causing the alarm. Would be great if it could be upgraded.
